drivers/can/can.c

```diff
@@ -89,6 +89,12 @@
  * Private Function Prototypes
  ****************************************************************************/
 
+/* CAN helpers */
+
+static uint8_t        can_dlc2bytes(uint8_t dlc);
+#if 0 /* Not used */
+static uint8_t        can_bytes2dlc(uint8_t nbytes);
+#endif
```

Comment:
Why is it not used? If it is not used, why to keep it in the source code?

Reply:
OK, these codes are changed by my teacher, I will tell him submit these codes to nuttx community and describe the reason about each topic.